### PR TITLE
Changes in project structure, updates to README

### LINKED ISSUE #22: Update readme with proper deploy and local testing instruction

### PR DESCRIPTION
**Pull request by petertrr**
### What's done:
* Removed save-core and save-plugins
* Changed main README.md
* Added notes on local deployment
* Added gradle task deployLocal
* Changed version format for docker images

Closes #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,131 +2,14 @@
...
-languages in the world like Python and [Java](https://stackoverflow.com/questions/6050618/is-there-a-java-equivalent-to-misra-c).
- 
-In this situation each and every new developer that reinvents his new code style or mechanism of static analysis each time reinvents his brand new test framework and writting test sets
-that have been written already thousands of times for his analyzer/linter. Someone uses existing guidelines like [Google code style](https://google.github.io/styleguide/javaguide.html)
-or using [PMD rules](https://pmd.github.io/). But in all cases lot of time will be spent on reinventing, writing and debuging tests.
-
 ## What is SAVE?
-SAVE - is an eco-system for measuring, testing and certification of static code analyzers. Instead of writing your test framework SAVE will provide you a command line application with a
+[SAVE](https://github.com/cqfn/save) - is an eco-system for measuring, testing and certification of static code analyzers. Instead of writing your test framework SAVE will provide you a command line application with a
```
Comment (aktsay6) [inline README.md:6]: I guess the link has changed to `https://github.com/cqfn/save-cloud`

File: save-deploy/README.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+## Server deployment
+* Ensure that docker daemon is running and that docker is in swarm mode.
+* Pull new changes to the server and run `./gradlew deployDockerStack`.
+
+## Local deployment
+* Ensure that docker daemon is running and docker-compose is installed.
+* Run `./gradlew deployLocal` to start only some components.
 
 ## Ports allocation
 * 5000 - save-backend
```
Comment (aktsay6) [inline save-deploy/README.md:24]: Do we need to add here that h2 console is on port 8081 in dev profile?

**Review by aktsay6 — CHANGES_REQUESTED** [review]
see the comments

File: README.md (hunk 1 above)
Comment (petertrr) [reply README.md:6]: Here it's intentionally a link to main repo with SAVE CLI, but I forgot to change links in badges, thanks for noticing.

File: save-deploy/README.md (hunk 2 above)
Comment (petertrr) [reply save-deploy/README.md:24]: Added

File: save-deploy/README.md [hunk 3]
```diff
@@ -1,10 +1,31 @@
...
 * 5100 - save-orchestrator
 * 5200 - save-test-preprocessor
 * 6000 - local docker registry (not used currently)
+* 8081 - h2 console (active with `dev` profile)
```
Comment (akuleshov7) [inline save-deploy/README.md:28]: create a table with ports - it is more friendly

**Review by akuleshov7 — APPROVED** [review]
lgtm

**Review by aktsay6 — APPROVED** [review]
lgtm

**Pull request merged**